makejdk.sh

```diff
@@ -215,6 +218,10 @@ else
   $WORKING_DIR/sbin/build.sh $WORKING_DIR $TARGET_DIR $OPENJDK_REPO_NAME $BUILD_FULL_NAME $JVM_VARIANT
 
   if [[ ! -z $JTREG ]]; then
-    $WORKING_DIR/sbin/jtreg.sh $WORKING_DIR $OPENJDK_REPO_NAME $BUILD_FULL_NAME
+    if [[ ! -z $JTREG_TEST_SUBSETS ]]; then
+      $WORKING_DIR/sbin/jtreg.sh $WORKING_DIR $OPENJDK_REPO_NAME $BUILD_FULL_NAME $JTREG_TEST_SUBSETS
```

Comment:
Currently, when we run the jtreg without options, we default to jdk_core and langtools (the latter doesn't seem to work, but jdk_core has enough coverage that we don't mind atm).

We could default to jdk_core, which gives us a core set of tests and removes the non-functional langtools, or we could switch to jdk_all. I don't know how long jdk_all takes, but I'm guessing it's longer than jdk_core. 

The latter already takes 25 minutes on our ultra-fast linux server.